common/classes/com/arjuna/common/internal/util/ClassloadingUtility.java

```diff
@@ -23,7 +23,7 @@ public class ClassloadingUtility
 {
     /**
      * Load a class. No instantiation.
-     * 
+     * <p>
```

Comment:
This (and the others) should go in a different commit (dedicated to a change of style)

Reply:
and...btw...shouldn't it be confined with a `<\p>`. Just asking for curiosity

Reply:
Oh okay then, bit of a nit pic but I'll remove them.
Note that empty elements do not require an explicit closing tag so I'll use `<p/>`.

Reply:
But something for others to note is that this is what Intellij added.

Reply:
Good to know :-)